inject-java/src/test/groovy/io/micronaut/visitors/PropertyElementSpec.groovy

```diff
@@ -278,19 +278,15 @@ class Parameters {
 
         def beanProperty = parameter.type.beanProperties.get(0)
         beanProperty.type.annotationNames.sort() == [
```

Comment:
This test had to be change becase `BeanProperties` is now added as a stereotype.